xonsh/prompt/vc.py

```diff
@@ -50,6 +51,8 @@ def get_git_branch():
     t.join(timeout=timeout)
     try:
         branch = q.get_nowait()
+        remove_ansi = re.compile(r'(?:\x1B[@-_]|[\x80-\x9F])[0-?]*[ -/]*[@-~]')
```

Comment:
This will compile the regex every time we hit a VC dir which we probably want to avoid.  

You can see an example of how to make it lazily evaluated here: https://github.com/xonsh/xonsh/blob/master/xonsh/built_ins.py#L179

Reply:
Done